shared_assets/shader/distance_field_sprite.pixel.metal

```diff
@@ -37,8 +43,7 @@ fragment float4 pixel_func (
 	float outline = 1.0 - smoothstep(inEdge - s, inEdge + s, a);
 	float4 colFill = v.colour;
 	float4 colOutline = material.outlineColour;
-	float4 col = mix(colFill, colOutline, outline);
```

Comment:
I think you can keep this line and remove the one with the `lerp` (and also the `lerp` function itself)